server/alerts_smtp.py

```diff
@@ -0,0 +1,66 @@
+import king_phisher.plugins as plugin_opts
+import king_phisher.server.plugins as plugins
+import king_phisher.server.signals as signals
+from os import environ
+
+
+try:
+	from smtp2go import core as smtp2go
+except ImportError:
+	has_smtp2go = False
+else:
+	has_smtp2go = True
+
+EXAMPLE_CONFIG = """\
+  api_key: <api_key>
+"""
+
+
+class Plugin(plugins.ServerPlugin):
+	authors = ['Spencer McIntyre', 'Mike Stringer']
+	title = 'Campaign Alerts: via SMTP2Go'
+	description = """
+	Send campaign alerts via the SMTP2go lib. This requires that users specify
+    their email through the King Phisher client to subscribe to notifications.
```

Comment:
This line should be aligned with `Send`. If it looks aligned in your editor, you may want to check if you're consistently using hard tabs instead of spaces.